CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - Fix: [#2379] Text cursor is not rendering in the right position.
 - Fix: [#2380] 'Number of smoothing passes' label is displayed incorrectly.
 - Fix: [#2396] Crash when creating a station of greater than 80 tiles.
+- Fix: [#2377] Error message appearing below window when cloning vehicle without having sufficient funds
```

Comment:
Could you move this so the issue numbers are ordered again?

I also prefer these lines to start with their premise, e.g. "When cloning a vehicle, ..."

Reply:
Fair enough, d7c8a9b